indigo/distribution.yaml

```diff
@@ -837,16 +837,16 @@ repositories:
     doc:
       type: git
       url: https://github.com/delftrobotics/camera_pose_calibration.git
-      version: master
+      version: indigo
     release:
       tags:
         release: release/indigo/{package}/{version}
       url: https://github.com/delftrobotics/camera_pose_calibration-release.git
-      version: 0.1.2-0
```

Comment:
The release repo does not have any 0.1.3 related branches and tags. You need to invoke `bloom-release` for that.